### PR TITLE
Update webhint

### PR DESCRIPTION
**Pull request by hkal**
This patch updates webhint to enable Code Fixes and address a few dependency related security issues.

### CODE REVIEW

**Comment by vidorteg** [comment]
we also need to update CHANGELOG.md to reflect the webhint update

File: package.json [hunk 1]
```diff
@@ -678,7 +678,7 @@
         "utf-8-validate": "5.0.6",
         "vscode-chrome-debug-core": "6.8.11",
         "vscode-extension-telemetry": "0.4.1",
-        "vscode-webhint": "2.1.2",
+        "vscode-webhint": "2.1.3",
```
Comment (vidorteg) [inline package.json:681]: We should also do a minor version bump on the extension itself. (line 5: 2.2.0)

**Review by vidorteg — CHANGES_REQUESTED** [review]
Left a couple of comments.

**Comment by hkal** [comment]
> we also need to update CHANGELOG.md to reflect the webhint update

This patch is just for updating webhint. Not the release patch for the vscode extension itself.

**Review by vidorteg — APPROVED** [review]
LGTM

**Pull request merged**